src/DependencyInjection/TacticianDomainEventExtension.php

```diff
@@ -40,32 +41,98 @@ private function addListeners(ContainerBuilder $container)
         $definition = $container->getDefinition('tactician_domain_events.dispatcher');
         $taggedServices = $container->findTaggedServiceIds('tactician.event_listener');
 
+        $listeners = array_merge(
+            $this->findListenersForEvent($taggedServices),
+            $this->findListenersForTypeHints($container, $taggedServices)
+        );
+
+        foreach($listeners as $item) {
+            if (!class_exists($item['event'])) {
+                throw new \Exception(
+                    sprintf(
+                        'Class %s registered as an event class in %s does not exist',
```

Comment:
Please add quotes: `Class "%s" registered as an event class in "%s" does not exis` for better debugging

Reply:
Ok
```suggestion
                        'Class "%s" registered as an event class in "%s" does not exist',
```